webextensions/api/sessions.json

```
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sessions",
           "support": {
             "chrome": {
-              "version_added": "≤58"
+              "version_added": "30"
```

:x: https://github.com/chromium/chromium/commit/10d71e33310e3e3ae5f9d9f185b1d9665df0e845 ([31.0.1609.0](https://github.com/chromium/chromium/releases/tag/31.0.1609.0) before 31.0.1650) added the feature to the "dev" channel (renamed from `sessionRestore`). It was only promoted to stable in https://github.com/chromium/chromium/commit/c19be7dd71d662c93eb91bae93bebadd6a52d1f6 ([37.0.2058.0](https://github.com/chromium/chromium/releases/tag/37.0.2058.0) before 37.0.2062).

```suggestion
              "version_added": "37"
```